virttest/qemu_devices/qcontainer.py

```diff
@@ -616,12 +616,11 @@ def simple_hotplug(self, device, monitor, bus=None):
                 if self.is_pci_device(device['driver']):
                     bus = self.get_buses({'aobject': 'pci.0'})[0]
                 if not isinstance(device.parent_bus, (list, tuple)):
-                    device.parent_bus = [device.parent_bus]
-                for parent_bus in device.parent_bus:
-                    for _bus in self.get_buses(parent_bus):
-                        if _bus.bus_item == 'bus':
-                            bus = _bus
-                            break
+                    for parent_bus in [device.parent_bus]:
```

Comment:
Why move this? If the bus is already a list or tuple, it wont get correct bus this way.

Reply:
According to `qcontainer.py`, line 620. 
```python
for parent_bus in device.parent_bus:
```
and we can see other devices info define by framework
```
device
  aid = idJo6csb
  aobject = None
  parent_bus = ({'aobject': 'pci.0'},)
  child_bus = []
  params:
    driver = virtio-net-pci
    mac = 9a:9e:9f:a0:a1:a2
    id = idJo6csb
    vectors = 4
    netdev = id9yY2co
    bus = pci.0
    addr = 0x4
```
If set `device.parent_bus = [device.parent_bus]`, the device info will be `parent_bus = [{'aobject': 'pci.0'}]` instead of `parent_bus = ({'aobject': 'pci.0'},)`
So use `for parent_bus in [device.parent_bus]` will be better.

Reply:
@PaulYuuu My point is, `if not isinstance(device.parent_bus, (list, tuple))` should not be a condition for `bus = _bus`. Both `[{'aobject': 'pci.0'}]` and `({'aobject': 'pci.0'},)` is acceptable format for `device.parent_bus`. Similar usage here: https://github.com/avocado-framework/avocado-vt/blob/master/virttest/qemu_devices/qcontainer.py#L494

Reply:
@PaulYuuu And what you get in the output, might get from here: https://github.com/avocado-framework/avocado-vt/blob/master/virttest/qemu_devices/qcontainer.py#L535 So we might modified same way if you want it be consistent. Thanks.